test/browser/test_sdl2_canvas_write.c

```diff
@@ -5,73 +5,75 @@
 
 #include <assert.h>
 #include <SDL.h>
```

Comment:
is this test change unrelated?

Reply:
Yeah, these are the sdl2 tests that I cloned in order to create the new sdl3 tests.  I cleaned up the formatting in these two tests before cloning them

Reply:
View with ignore-whitespace to see that they are just cleanups.